openfisca_us/microdata/tests/cps/taxcalc_cps.yml

```diff
@@ -0,0 +1,136 @@
+DSI:
+  2020: 189927269.0
+EIC:
+  2020: 9228312693.0
+FLPDYR:
+  2020: 41355857382607.0
+MARS:
+  2020: 34571710317.0
+RECID:
+  2020: 2868868782968629.0
+XTOT:
+  2020: 38912156785.0
+a_lineno:
+  2020: 32607341954.0
+age_head:
+  2020: 961076447150.0
+age_spouse:
+  2020: 296862623734.0
+agi_bin:
+  2020: 122252399747.0
+blind_head:
+  2020: 455822086.0
+blind_spouse:
+  2020: 79696777.0
+e00200:
+  2020: 728084861887664.0
```

Comment:
BTW I think these are off by 100x, this is $728 trillion. Weights are provided with 100x multipliers to avoid decimals.

Reply:
Ah right, yep removed for now